python/cuml/fil/fil.pyx

```diff
@@ -315,14 +316,16 @@ cdef class ForestInference_impl():
                                         bool output_class,
                                         str algo,
                                         float threshold,
-                                        str storage_type):
+                                        str storage_type,
+                                        int blocks_per_sm):
```

Comment:
I've kept the Cython API changes (ForestInference_impl) because nobody is using (or supposed to use) this class directly. No point editing the same lines over and over.